tests/scripts/multus/host-cfg-ds.yaml

```diff
@@ -29,7 +29,7 @@ spec:
       terminationGracePeriodSeconds: 0 # allow updating/deleting immediately
       containers:
         - name: test
-          image: quay.io/ceph/ceph:v18
+          image: jonlabelle/network-tools
```

Comment:
Is there any more official image? For example, just use an ubuntu image and install network tools if missing? Or maybe just add a comment about this image and an alternate if this one goes away?

Reply:
That's a good point. Busybox should be good for this since we really just need `ip`. I use this network tools image for doing DNS debugging, but we don't need all that here

Reply:
I really don't know what it is about the busybox image, but it's not working. It's causing some kind of network interference on the node where the validation nginx server is running. It looks like busybox runs as the nginx user, but even when I make sure this pod runs as root, it still causes the interference. Everything looks technically correct, but something isn't working, and I'm not sure why.

This isn't something that I would ever plan to encourage users to run in prod, and I think the jonlabelle image is likely to be fairly stable. In the interest of not getting caught in the weeds more, I'd like to merge this.

Reply:
Ok sounds good to merge it then, not a big deal since it is just the CI